composer.json

```diff
@@ -15,28 +15,38 @@
             "homepage": "https://github.com/sonata-project/SonataCoreBundle/contributors"
         }
     ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/Ener-Getick/PHP-CS-Fixer"
+        },
```

Comment:
Must be removed once https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1510 is merged.